### ISSUE TITLE
Facebook Graph API v2.8 EOL 18th April 2019

### ISSUE THREAD

**Issue by coling** [hybridauth/hybridauth#1056]
From [Facebook documentation](https://developers.facebook.com/docs/graph-api/changelog/version2.8/), it states that the Graph API v2.8 is only available until April 18, 2019 which is fast approaching.

The code in Hybrid Auth 2.13.0 currently has this version [hard coded](https://github.com/hybridauth/hybridauth/blob/v2/hybridauth/Hybrid/Providers/Facebook.php#L62) (it's also hard coded in the master branch for photo URLs but this is perhaps less important or just a coincidence)

Is this something that should be updated accordingly before the deadline?

**Comment by ApacheEx** [comment]
good catch, I think we are safe to do it in the following way:

for v2: 

```php
'default_graph_version' => !empty($this->config['graph_version']) $this->config['graph_version'] : 'v2.12',
```

and for v3 - similar logic.

I will prepare a PR today and release a new version.

**Comment by ApacheEx** [comment]
released as v2.14
